packages/@aws-cdk-testing/framework-integ/test/aws-ec2/test/integ.vpc-dual-stack-ec2.assets/web-server.py

```diff
@@ -0,0 +1,42 @@
+from http.server import SimpleHTTPRequestHandler
+import urllib.request
+import json
+
+class MyRequestHandler(SimpleHTTPRequestHandler):
+    def do_GET(self):
+        if self.path == '/':
+            self.send_response(200)
+            self.send_header('Content-type', 'application/json')
+            self.end_headers()
+
+            try:
+                # IPv4 request
+                response = urllib.request.urlopen('http://www.google.com')
```

Comment:
Change to `http://ipv4.google.com`

Reply:
:+1: ipv4.google.com would also be useful for testing NAT64

Reply:
I'd be concerned to see a test that is reliant on any particular service out there, particularly a third party one.